docs/experiments-vector-stores.md

```diff
@@ -0,0 +1,286 @@
+# BGE-base for NFCorpus in Database Vector Stores
+This guide contains instructions for running a BGE-base baseline for NFCorpus in the following databases:
+
+- [DuckDB](#duckdb)
+- [ChromaDB](#chromadb)
+- [Weaviate](#weaviate)
+
+## Indexing
+Assuming you have completed [this guide](https://github.com/castorini/pyserini/blob/master/docs/experiments-nfcorpus.md) and fetched the data, we start by encoding the corpus and queries to obtain embeddings. We will feed these embeddings into the vector stores directly. 
+
```

Comment:
as a convention, we do each sentence on a separate line - this allows us to read diffs easily.